lazer/solana/src/bin/client.rs

```diff
@@ -49,6 +52,14 @@ fn main() -> anyhow::Result<()> {
         let signature = client.send_and_confirm_transaction(&tx)?;
         println!("OK {signature:?}");
     } else if cmd == "update" {
+        let state_data = client.get_account_data(&data_pda_key)?;
+        let state = from_bytes::<State>(&state_data);
+        println!("state: {state:?}");
```

Comment:
What kinds of things are in state that we would want to print it out? I wonder if we can migrate to using tracing crate for logging to change visibility of some of these print statements.